bin/lcov

```diff
@@ -4291,6 +4296,17 @@ sub print_overall_rate($$$$$$$$$)
 	info("  branches...: %s\n",
 	     get_overall_line($br_found, $br_hit, "branch", "branches"))
 		if ($br_do);
+
+	if ($ln_do && $fail_under_lines > 0) {
+		my $actual_rate = ($ln_hit / $ln_found);
+		my $expected_rate = $fail_under_lines / 100;
+		if ($actual_rate ge $expected_rate) {
```

Comment:
`ge` is a string comparison operator. Use `>=` instead.

Reply:
Ah, fixed.